webapp/loggedin/modules/data/dataVis/dataQuery/components/tableRows.js

```diff
@@ -34,9 +34,7 @@ const TableRows = ({ nodeDefCols, data, offset, colWidth, defaultColWidth, editM
             const validation = Record.getValidation(record)
 
             return (
-              <div key={String(i)} className="table__row">
-                <ErrorBadge validation={validation} showLabel={false} className="error-badge-inverse" />
-
+              <div key={String(i)} className={`table__row${Validation.isValid(validation) ? '' : ' invalid'}`}>
```

Comment:
does this make all row red? if yes, only invalid cell should be red

Reply:
ok